### PR TITLE
Added support for nikic/php-parser > 4.6

### PR DESCRIPTION
**Pull request by xfuturomax**

### CODE REVIEW

**Comment by lisachenko** [comment]
Hello! Thank you for this PR! I think it should be next major version to avoid installation of php-parser > 4.6 again to existing projects.

File: src/ReflectionEngine.php [hunk 1]
```diff
@@ -281,4 +281,8 @@ public static function parseFileNamespace($fileName, $namespaceName)
         throw new ReflectionException("Namespace $namespaceName was not found in the file $fileName");
     }
 
+    public static function getLexer()
```
Comment (lisachenko) [inline src/ReflectionEngine.php:284]: Please add a return type declaration for this method

**Review by lisachenko — CHANGES_REQUESTED** [review]

File: src/ReflectionEngine.php (hunk 1 above)
Comment (xfuturomax) [reply src/ReflectionEngine.php]: Done

**Review by lisachenko — APPROVED** [review]

**Comment by alekc** [comment]
Hi @lisachenko , do you have an approximate frame for the next version (as in when it's going to hit mainstream without having to use hacks in composer json)? 
Cheers

**Pull request merged**